src/base/static/state/ducks/ui.js

```diff
@@ -31,13 +35,17 @@ export function setRightSidebar(isExpanded) {
 export function setLeftSidebar(status) {
   return { type: SET_UI_LEFT_SIDEBAR, payload: status };
 }
+export function setCurrentTemplate(templateName) {
+  return { type: SET_CURRENT_TEMPLATE, payload: templateName };
+}
 
 // Reducers:
 const INITIAL_STATE = {
   isContentPanelOpen: undefined,
   isSidebarExpanded: undefined,
   isLeftSidebarExpanded: false,
   leftSidebarComponent: undefined,
+  currentTemplate: "map",
```

Comment:
The "currentTemplate" is assumed to be either `map` or `list`, but this could also be `dashboard` or `story` or other things down the road.